pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -376,8 +376,15 @@ class AnalyticsImpl implements Analytics {
   }
 
   @override
-  String get getConsentMessage =>
-      kToolsMessage.replaceAll('[tool name]', tool.description);
+  String get getConsentMessage {
+    // The command to swap in the consent message
+    final String commandString =
+        tool == DashTool.flutterTool ? 'flutter' : 'dart';
+
+    return kToolsMessage
+        .replaceAll('[tool name]', tool.description)
+        .replaceAll('[dart|flutter]', commandString);
+  }
```

Comment:
This has been added to address issue https://github.com/dart-lang/tools/issues/82 to change the consent message's command based on which tool is using it